test/ref/continuous/johnsonsu.R

```diff
@@ -0,0 +1,34 @@
+library("ExtDist")
+
+JohnsonSU <- R6Class("JohnsonSU",
+	inherit = ContinuousDistribution,
+	public = list(names = c("xi", "lambda", "gamma", "delta"),
+		xi = NA,
+		lambda = NA,
+		gamma = NA,
+		delta = NA,
+		initialize = function(xi = 0, lambda = 1, gamma = 0, delta = 1) {
+			self$xi <- xi
+			self$lambda <- lambda
+			self$gamma <- gamma
+			self$delta <- delta
+		},
+		supp = function() { c(-Inf, Inf) },
+		properties = function() { list() },
+		pdf = function(x, log = FALSE) {
+			p <- dJohnsonSU(x, xi = self$xi, lambda = self$lambda, gamma = self$gamma, delta = self$delta)
+            if (log) {
```

Comment:
It looks like this file has mixed tabs and spaces. You could do a find and replace for changing each \t to four spaces.

Reply:
Will do!